mtp_noms_ops/urls.py

```diff
@@ -88,6 +89,8 @@ def root_view(request):
     ),
     url(r'^', include('security.urls', namespace='security')),
 
+    # Override mtp_common.user_admin's /users/new/ view
+    url(r'^users/new/$', UserCreationView.as_view(), name='new-user'),
```

Comment:
Hmm, I wonder whether it would be nicer to leave defining the `new-user` url down to the service, so there's no overriding needed. What do you think?

Reply:
Not sure, it makes sense for `mtp-common` to provide this functionality but I had to override some of copy, e.g. the checkbox label:

> FIU This account is for an FIU user (FIU users will be able to manage other user accounts)

I think it's nice that `mtp-common` provides a good enough/generic implementation instead of forcing the service to re-implement it.